frontend/src/modules/member/components/view/_aside/_aside-identities.vue

```diff
@@ -42,11 +42,16 @@
           target="_blank"
           rel="noopener noreferrer"
         >
-          <div class="flex gap-3 items-center">
-            <app-platform platform="email" />
-            <span class="text-gray-900 text-xs">
-              {{ email }}</span>
-          </div>
+          <el-tooltip
+            placement="top"
+            :content="email"
```

Comment:
Also, can you disable the tooltip if the text is not truncated? There's an example in `organization-name.vue` component